src/AppInstallerRepositoryCore/SourceList.cpp

```diff
@@ -207,6 +209,16 @@ namespace AppInstaller::Repository
         {
             return details.IsTombstone || details.Origin == SourceOrigin::Metadata || !details.IsVisible;
         }
+
+        std::string_view GetWingetCommunitySource()
+        {
+            if (Settings::Experiment::GetState(Settings::Experiment::Key::CDN).IsEnabled())
+            {
+                return s_Source_WingetCommunityExperimental_Arg;
+            }
+
+            return s_Source_WingetCommunityDefault_Arg;
+        }

```

Comment:
We do some validation when a user tries to add one of the default sources. For example, to block it if the group policy restricts the default source. This may allow for an unintended way to bypass that validation. E.g., somebody has the experiment and the group policy, which means we block adding the cdn2.winget... source, but they can still add cdn.winget...

Reply:
Yes, I think the following should cover it:

1. Remove `GetWellKnownSourceArg`
2. Update `CheckForWellKnownSourceMatch` to
  a. be case insensitive
  b. handle both DNS entries
  c. allow for not checking certain values (ex. name), maybe by passing an empty `string_view`
4. Replace use of `GetWellKnownSourceArg` in `GetPolicyBlockingUserSource` with the updated `CheckForWellKnownSourceMatch`

It didn't look like any other paths were relevant there.